### ISSUE TITLE
Cannot edit office365 information after provisioning a into a customer

### ISSUE THREAD

**Issue by kazimdshafiullah** [socfortress/CoPilot#268]
Hello everyone! I want to change the information like tenant id or client secret after office365 provisioning to the customer? I have accidentally mistake one of information while provisioning office365 and cannot change the input value. Even cannot delete the values.

**Comment by taylorwalton** [comment]
@kazimdshafiullah unfortunately, you cannot edit the office365 details after provisioning has been done. You will need to make two manual modifications. 1. modify the `office365` block within the `ossec.conf` of the Wazuh Maanger (make sure to restart service). 2. change the Stream rule within Graylog to match on the correct Tenant ID value

**Comment by kazimdshafiullah** [comment]
As I have misplaced already while input the office365 details, I got error while deploy after saving the values. Error is below. As the deploy is not completed there is no configuration in ossec.conf and Graylog stream rule as well.
Error:
![Screenshot 2024-07-04 234255](https://github.com/socfortress/CoPilot/assets/107104529/875d9098-c604-4786-84a1-b33cc141804a)

**Comment by kazimdshafiullah** [comment]
Also I have checked log from backend container. If I don't configure office365 manually in wazuh manager then I found this error.
`2024-07-09 10:03:55.576 | ERROR    | app.db.db_session:get_db_session:185 - Error during DB session: junk after document element: line 366, column 0`

And if I configure office365 manually in wazuh manager then I found this error.
`2024-07-09 10:01:23.350 | ERROR    | app.db.db_session:get_db_session:185 - Error during DB session`
